app/uploaders/spotlight/configurable_uploader_versions.rb

```diff
@@ -19,7 +19,7 @@ def configured_versions
         },
         {
           version: :square,
-          blacklight_config_field: :square_image_field,
+          field: Spotlight::Engine.config.square_image_field,
```

Comment:
Okay, I tried something.  Seems a little cleaner than a conditional appending configurations to the array so I'm rejecting any nil fields.  Let me know if you had something better in mind.